src/util/evented.js

```diff
@@ -155,7 +155,7 @@ export class Evented {
             (this._listeners && this._listeners[type] && this._listeners[type].length > 0) ||
             (this._oneTimeListeners && this._oneTimeListeners[type] && this._oneTimeListeners[type].length > 0) ||
             (this._eventedParent && this._eventedParent.listens(type))
-        );
+        ) === true;
```

Comment:
It's a little easier and clearer to do `return !!(...` instead. Transform the `undefined` value into a boolean.

While we're making a change to this file, we might as well clean up the grammar in the docs comment too.

`Returns a true if this instance of Evented or any forwardeed instances of Evented have a listener for the specified type.` should be `Returns true if this instance of Evented or any forwarded instances of Evented have a listener for the specified type.`